Index: sourcecode/test/BoardTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package sourcecode.test;\n\npublic class BoardTest {\n}\n
===================================================================
diff --git a/sourcecode/test/BoardTest.java b/sourcecode/test/BoardTest.java
--- a/sourcecode/test/BoardTest.java	
+++ b/sourcecode/test/BoardTest.java	
@@ -1,4 +1,16 @@
 package sourcecode.test;
 
+import sourcecode.board.Board;
+import sourcecode.board.Cell;
+
 public class BoardTest {
+    public static void main(String[] args) {
+        Board newBoard = new Board();
+
+        Cell[] board = newBoard.getBoard();
+        for(Cell cell:board) {
+            System.out.println(cell);
+        }
+
+    }
 }
